python/tvm/hybrid/runtime.py

```diff
@@ -36,6 +36,22 @@ def allocate(shape, dtype='float32', scope='global'): #pylint: disable=unused-ar
     return numpy.zeros(shape).astype(dtype)
 
 
+def rsqrt(x):
+    """
+    Computes reciprocal of square root of x element-wise
+
+    Parameters
+    ----------
+    x: a real number
```

Comment:
Is this actually a real number, and not a tensor?

Reply:
This is tensor, thanks.